app/models/phrase.rb

```diff
@@ -0,0 +1,4 @@
+class Phrase < ApplicationRecord
+  has_many :survey_phrases
```

Comment:
I think it's good practice to be explicit [with the `dependent` argument](https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/has_many) which specifies what to do with the `survey_phrases` if the `phrase` gets deleted. I figured in this case `dependant: :destroy` is reasonable.

Reply:
That's a fair point. In the app as it stands, nothing is ever deleted - new feedback will only either create new phrases or reference existing ones. From the perspective of good practice I agree with you - I'll do this now.